migrate_passwords.py

```diff
@@ -0,0 +1,62 @@
+''' Python script to migrate database password hashing algorithms. '''
```

Comment:
Cool, sounds good. Didn't know you were planning to delete it. In that case, it's fine as is.